tor/strings/responses.py

```diff
@@ -126,6 +126,22 @@
     'The Mods of /r/TranscribersOfReddit'
 )
 
+dm_subject = (
+    'Direct Message'
+)
+
+dm_body = (
+    'Hi there! Thanks for messaging me!'
+    '\n\n'
+    'Unfortunately I\'m a bot and so I can\'t help you out myself. I\'ve '
+    'forwarded your message to a moderator so they can help you out! They '
```

Comment:
Let's change this to "forwarded your message to the moderators" -- don't want people to get confused and start PM'ing us directly trying to see if we got their message.